File: 14491.py
T = int(input())

nine = [0 for i in range(5)]
base = 6561; index = 4
while T > 0:
    if T >= base:
        nine[index] = T // base
        T %= base

    base //= 9
    index -= 1

index = 4
while not nine[index]: index -= 1
for i in range(index, -1, -1):
    print(nine[i], end='')
print()

File: 14491.cpp
#include <iostream>
using namespace std;

int main() {
	int T; cin >> T;

	int nine[5] = { 0, };
	int base = 6561, index = 4;
	while (T > 0) {
		if (T >= base) {
			nine[index] = T / base;
			T %= base;
		}

		base /= 9;
		index--;
	}

	index = 4;
	while (!nine[index]) index--;
	for (; index >= 0; index--)
		cout << nine[index];
	cout << endl;
}
